manually-managed/README.md

```diff
@@ -0,0 +1,33 @@
+# Manually Managed Services
+
+While most of our infrastructure is managed with Kubernetes, several stateful services and other components are managed separately. At current, these are managed manually, but we are considering the best tools for this, e.g. Ansible. (If you have expertise or time to contribute here, let us know in the issues or on Slack!)
+
+
+## Redis Cache
+
+We use Redis for caching and for queuing in Rails (that is, in `web-monitoring-db`). Our Redis caching instance is currently hand-managed as an EC2 machine on AWS. Its current configuration (excepting the password) can be found in [`redis-cache.conf`](./redis-cache.conf). It also has an elastic IP.
```

Comment:
Let's link `web-monitoring-db` to the repo: https://github.com/edgi-govdata-archiving/web-monitoring-db. 

I'm a lazy bum and and happy to have just one spot on the page that goes to the repo, but I can see wanting to link all other instances of `web-monitoring=db` as well.